dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 git+git://github.com/plotly/dash@master#egg=dash
+git+git://github.com/plotly/dash-core-components@fix-tests-in-renderer#egg=dash_core_components
```

Comment:
hmm, who's going to win, this version or the numbered version coming from dash?

Eventually we probably want to move to have CI explicitly clone this (and the other core repos) then build it and `install -e` it, then there's no ambiguity (also we're not dependent on committing the build to the repo 🎉 )

Reply:
Did not think this through. And the `pip list | grep dash` line has been removed!

Reply:
And that last change won't tell me anything as I left the version number as is